coffee_github/src/repository.rs

```diff
@@ -36,6 +36,8 @@ pub struct Github {
     /// the name of the branch to be able to
     /// pull the changes from the correct branch
     branch: String,
+    /// the commit id of the last commit
+    commit: String,
```

Comment:
```suggestion
    commit: Option<String>,
```

When we are restoring the object from the database we should not crash (if someone is using coffee before this change) so we use optional and then after a while we change it is required (put a commit on it as a FIXME).